andela/troupon
# #126489161 Add Ratings & Reviews

#### What does this PR do?

This PR allows users to add a rating and review for deals that they purchase.
#### Description of Task to be completed?
1. Allow users to add a rating
2. Allow users to add a review
3. Ensure users can only rate and review deals they have purchased
4. Ensure users can only leave one rating and review per deal
#### How should this be manually tested?

While on the homepage or the Deals page, click on the More Details button on any deal. This should direct you to the individual deals page. If there are existing reviews for the deal, they should be displayed as well as the average rating. If you are logged in and have purchased the deal, you should be able to see a form containing the stars for rating and a text area. Filling this form and clicking submit allows you to leave a review. 

If you are not logged in, you should instead see a button directing you to the login page. If you have not purchased the deal, you should see a message informing you that you must purchase it in order to leave a review.
#### What are the relevant pivotal tracker stories?
#126489161
#### Screenshots

<img width="1237" alt="screen shot 2016-08-08 at 09 37 29" src="https://cloud.githubusercontent.com/assets/18420962/17508238/a4d3a39a-5e1c-11e6-8071-7bd092128892.png">
<img width="1236" alt="screen shot 2016-08-08 at 09 39 10" src="https://cloud.githubusercontent.com/assets/18420962/17508248/b3abf5f2-5e1c-11e6-908b-8e976cb48eb4.png">
<img width="1232" alt="screen shot 2016-08-09 at 10 29 45" src="https://cloud.githubusercontent.com/assets/18420962/17508255/b7f0a4b4-5e1c-11e6-93fa-c77c19df2b0c.png">
